docs/Server-Methods.md

```diff
@@ -143,6 +143,14 @@ The logger instance, check [here](https://github.com/fastify/fastify/blob/master
 #### inject
 Fake http injection (for testing purposes) [here](https://github.com/fastify/fastify/blob/master/docs/Testing.md#inject).
 
+<a name="add-schema"></a>
+#### addSchema
+`fastify.addSchema(schemaObj)`, adds a shared schema to the Fastify instance. This allows you to reuse it everywhere in your application just by writing the schema id that you need.<br/>
+To learn more, see [shared schema example](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#shared-schema) in the [Validation and Serialization](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md) documentation.
+
+
+You can see [here](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#shared-schema) how it works.
```

Comment:
This sentence is redundant. It should be removed.

Reply:
Oooops, forgot to delete, sorry :P